pkg/controllers/cloud/cloudentityselector_controller.go

```diff
@@ -188,7 +189,7 @@ func (r *CloudEntitySelectorReconciler) addAccountPoller(selector *cloudv1alpha1
 	}
 	account := &cloudv1alpha1.CloudProviderAccount{}
 	_ = r.Get(context.TODO(), *accountNamespacedName, account)
-	accountCloudType, err := account.GetAccountProviderType()
+	accountCloudType, err := webhook.GetAccountProviderType(account)
```

Comment:
why is GetAccountProviderType in webhook package? doesnt seem right.. we need to move to common place

Reply:
Moved it under package utils. Renamed vm.go file in package utils to helpers.go and added GetAccountProviderType() in it.